Modules/_xxtestfuzz/fuzzer.c

```diff
@@ -142,6 +142,74 @@ static int fuzz_struct_unpack(const char* data, size_t size) {
 }
 
 
+PyObject* pickle_loads_method = NULL;
```

Comment:
Shouldn't you make all those `static`? Though I see other similar declarations in this file are not.

Reply:
there's definitely refactoring that could be done to this test module, rather than static I think we'd want to use the modern non-global module state APIs.  PEPs 489/587/573.  Lets not worry about that in this PR.